assets/js/opensuse-theme.js

```diff
@@ -133,8 +133,9 @@ function osMoreInformation(os) {
 
   moreInfoOpened = true
   //get the height of the main container
-  containerHeight = $('#opensuse-os').outerHeight()
-
+  containerHeight = $('#opensuse-os').outerHeight();
+  window.location.hash = '0';
```

Comment:
I dont really like this solution because it adds a hashtag #0 in the URL, which means nothing at all for a user.

I was gonna propose to add it as:

window.location.hash = title; (where titles is the name of the OS selected), so then the URL would change to: 

http://localhost:8000/#Leap

or 

http://localhost:8000/#Tumbleweed

And I like it more, but not 100%. 
If we decide to go in that direction, what I would recommend then, is that we detect when the URL has a hash, and if it does, then we automatically open the OS info with:

osMoreInformation(HashtagURL)

Otherwise, lets just detect the keydown and if it is backspace then we close the information.

@bisht-richa You decide how to go for now. If you prefer to go for detecting the event on keydown, then I can open a new Issue with that idea that just came up to have URL for users to share.